KomaMRICore/src/KomaMRICore.jl

```diff
@@ -0,0 +1,79 @@
+module KomaMRICore
+
+#IMPORT PACKAGES
+import Base.*, Base.+, Base.-, Base./, Base.vcat, Base.size,
+       Base.copy, Base.Threads.@spawn, Base.Threads.@threads,
+       Base.angle, Base.abs, Base.getproperty, Base.one, Base.zero
+#General
+using Random, Reexport, ThreadsX
+@reexport using Pkg
+#Printing
+using Scanf, ProgressMeter
+#Datatypes
+using Parameters
+#Simulation
+@reexport using CUDA, Interpolations
```

Comment:
Do we really need to reexport CUDA and Interpolations?

Reply:
They will not be reexported!

Reply:
We talked about CUDA being used by the default parameters for the UI, which was changed, but I believe we still have the  `@reexport using CUDA`. 

Sidenote: The `@info "Loading GPUs"` and `KomaMRICore.print_gpus()` will be printed with or without a CUDA-capable GPU (before it had an if statement). Not completely sure what the message says on a PC without NVIDIA GPUs, hopefully, "0 CUDA capable device(s).".